moveit_py/moveit/servo_client/teleop.py

```diff
@@ -52,12 +52,16 @@ class TeleopDevice(ABC, Node):
     This class expects both the `publish_command` and `record` methods to be overridden by inheriting classes.
     """
 
-    def __init__(self, node_name, device_name, device_config, ee_frame_name):
+    def __init__(self, node_name, device_name, device_config):
         super().__init__(node_name=node_name)
         self.device_name = device_name
         self.device_config = device_config
-        self.ee_frame_name = ee_frame_name
-
+        self.declare_parameters(
+            namespace=node_name, parameters=[("ee_frame_name", "panda_link8")]
```

Comment:
I don't think that we should have a hardcoded panda link in here. Can we just make this a required launch parameter?

Reply:
Good idea, is there an existing launch file that I can make changes to? or do I have create one for this purpose? @peterdavidfagan

Reply:
The servo sections of the following [file](https://github.com/ros-planning/moveit2_tutorials/blob/main/doc/examples/jupyter_notebook_prototyping/launch/jupyter_notebook_prototyping.launch.py) would be a good place to start.